internal/git/git.go

```
@@ -714,46 +714,36 @@ func (d *Driver) AddWorktree(root, sessionName string) (string, string, error) {
-			return "", "", err
+			return "", err
 		}
-		return path, branch, nil
+		return branch, nil
 	}
 	if _, err := d.run(root, "rev-parse", "--verify", "--quiet", "refs/heads/"+branch); err != nil {
-		return path, branch, nil
-	}
-	if _, err := os.Stat(newPath); err == nil {
-		return "", "", fmt.Errorf("worktree path already exists: %s", newPath)
+		return branch, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 737) but it returns nil

(nilerr)